src/notification.cpp

```diff
@@ -257,24 +257,75 @@ class NotificationPrivate : public NotificationData
 };
 
 /*!
+    \qmlmodule org.nemomobile.notifications
```

Comment:
Is this still necessary? I think we tried adding it to ensure qml-org-nemomobile-notifications-notification.html was generated but maybe it's not needed now. This line generates org-nemomobile-notifications-qmlmodule.html as an index for this module but nemo-qml-plugin-notifications.qdoc already provides its own index list.